src/open_inwoner/openzaak/api_models.py

```diff
@@ -273,3 +273,32 @@ class Notification(Model):
     actie: str
     aanmaakdatum: datetime
     kenmerken: Dict = field(default_factory=dict)
+
+
+@dataclass
+class OpenSubmission(Model):
+    url: str
```

Comment:
Now that open submissions are recognized as first-class citizens, do we still the `UniformCase` protocol that I introduced for the `InnerCaseListView`? It was a workaround at a time where it wasn't quite clear how we want to think about the distinction between zaken and open submissions:

- [https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/openzaak/types.py](https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/openzaak/types.py)
- [https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/cms/cases/views/cases.py#L89](https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/cms/cases/views/cases.py#L89)